gatsby/src/pages/faq.js

```diff
@@ -174,7 +174,7 @@ const Faq = ({ data }) => {
               </p>
               <p>
                 No, it’s nothing to do with the film (although you could go and
-                build virtual worlds on top of Matrix if you wanted :)
+                build virtual worlds on top of Matrix if you wanted :)).
```

Comment:
Old school smilies don’t get along too well with parentheses. Let’s make at an emoji. It’s 2021 after all. And since we’re talking about the Matrix movie, I’m suggesting sunglasses to fit the mood.

```suggestion
                build virtual worlds on top of Matrix if you wanted 😎).
```